build/yamls/base/conf/antrea-agent.conf

```diff
@@ -31,8 +31,9 @@ featureGates:
 # Enable controlling SNAT IPs of Pod egress traffic.
 #  Egress: false
 
-# Enable flexible IPAM mode for Antrea. This mode allows to assign IP Ranges to Namespaces,
-# Deployments and StatefulSets via IP Pool annotation.
+# Enable AntreaIPAM. AntreaIPAM is required by the bridging mode, and can assign IPs to bridging
```

Comment:
maybe:
> can assign IPs to Pods connected in bridging mode

BTW, is "bridging mode" a standard term and does it mean something different from "bridge mode"?

Reply:
As I said in the last comment, I do not have a good idea on how to name this mode/parameter. I call it bridging mode, just because the datapath behavior is like L2 bridging. And I welcome suggestions for a better name.

I think "bridge" vs "bridging" is like "switch" vs "switching".